Documentation/ceph-upgrade.md

```diff
@@ -39,6 +40,16 @@ those releases.
 * [Upgrade 0.6 to 0.7](https://rook.io/docs/rook/v0.7/upgrade.html)
 * [Upgrade 0.5 to 0.6](https://rook.io/docs/rook/v0.6/upgrade.html)
 
+## Important changes in this release
+* The minimum Kubernetes version has changed to v1.16. You must update to at least Kubernetes version
+v1.16 before upgrading Rook from v1.7 to v1.8.
+* Rook v1.8 no longer supports Ceph Nautilus (14.2.x). Nautilus users must upgrade Ceph to
```

Comment:
Do we have a doc we could point to on how to upgrade the Ceph cluster?

Reply:
We could link to the ceph upgrade section below 👍